apis/externalsecrets/v1beta1/clusterexternalsecret_types.go

```diff
@@ -93,9 +93,8 @@ type ClusterExternalSecretStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:resource:scope=Cluster,categories={externalsecrets},shortName=ces
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="Store",type=string,JSONPath=`.spec.secretStoreRef.name`
-// +kubebuilder:printcolumn:name="Refresh Interval",type=string,JSONPath=`.spec.refreshInterval`
-// +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[?(@.type=="Ready")].reason`
```

Comment:
I'm not too sure if we should keep the status column since IIUC, we don't have a corresponding field to the column, so I simply removed it.

Reply:
totally right, that doesn't belong here!